<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>561</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>479</width>
       <height>559</height>
      </rect>
     </property>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>551</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuRotation">
     <property name="title">
      <string>Rotation</string>
     </property>
     <addaction name="action90D"/>
     <addaction name="action90G"/>
     <addaction name="action180"/>
    </widget>
    <addaction name="actionFitToWindowAct"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="menuRotation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="Outils">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>553</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Outils</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="Outils_2">
    <property name="minimumSize">
     <size>
      <width>73</width>
      <height>0</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="Grey">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Grey</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Crop">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Crop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Pipette">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Pipette</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Histogramme">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Histogramme</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Flou">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Flou</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="Option">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>553</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="Option_2">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QTextBrowser" name="AffInfo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>141</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFitToWindowAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>fitToWindowAct</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action90D">
   <property name="text">
    <string>90° D</string>
   </property>
  </action>
  <action name="action180">
   <property name="text">
    <string>180°</string>
   </property>
  </action>
  <action name="action90G">
   <property name="text">
    <string>90° G</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
